raw_data/preprocessed/africa/sierra_leone_english.txt

```diff
@@ -0,0 +1,132 @@
+
+Notification to the Minister.
+(36) [1] Where a vessel or on shore or offshore facility is used or is involved in the commission of an offence against subsection [5] of the Minister of section 35 the owner or operator of such vessel or onshore or offshore facility shall: [a] promptly notify the Minister or other relevant authority of the discharge; [b] take all possible measures in accordance with standard removal methods to minimize pollution; [c] promptly comply with such other directions as the Minister may from time to time prescribe. [2] An owner or operator who fails to comply with subsection [1] commits an offence and is liable on conviction to a fine not exceeding five million leones or to a term of imprisonment not exceeding two years or to both the fine and imprisonment. 
+
+Legal proceedings.
+(37) Where any offence has been committed by a company or by proceedings any member of a partnership, firm or business, every Director, Manager, Secretary or other officer of that company directly connected with the project or any other member of the partnership or other person connected with the management of such firm or business shall be liable for such offence unless he proves to the satisfaction of the Court that: [a] he used all due diligence to secure compliance with the Act; and [b] such offence was committed without his knowledge, consent or connivance. 
+
+PART IV - THE NATIONAL ENVIRONMENT FUND
+
+Establishment of the National Environment Fund.
+(38) [1] There is hereby established a Fund to be known as the of National Environment Fund (hereafter referred to as "the Fund"). [2] The Fund shall consist of: [a] all monies provided by Government; and [b] donations, grants or bequests. [3] The Fund shall only be applied for the purposes of its objectives. [4] The Fund shall be managed and controlled by the Director who shall for this purpose be accountable to the Minister. [5] Notwithstanding subsection [4], no money shall be withdrawn from the Fund except under authority of a warrant issued by the Minister responsible for Finance.
```

Comment:
Please replace `as the of National Environment` with `as the National Environment`.